openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -122,3 +175,34 @@ func (configNodeV1) ValidateStatusUpdate(_ context.Context, uncastObj runtime.Ob
 
 	return errs
 }
+
+type validateCustomResourceWithNodeLister struct {
```

Comment:
I don't understand why we need this type. Why not to expose `SetExternalKubeInformerFactory` directly on `configNodeV1` ?

Reply:
I tried that and it wasn't registering right. This follows the convention of other objects like this so I assume there's a reason I don't fully understand

Reply:
I have a couple of suggestions for the admission plugin, but the authorizer looks good to me. I think we can merge it. Could you move the admission change to a separate PR?

Reply:
dropped the admission changes here